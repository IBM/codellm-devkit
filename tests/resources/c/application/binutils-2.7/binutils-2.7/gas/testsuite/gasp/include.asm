	HI
	.INCLUDE	"INC1.H"
	THERE
	.END
