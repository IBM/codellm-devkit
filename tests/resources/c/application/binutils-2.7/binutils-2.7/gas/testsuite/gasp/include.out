!	HI
	HI
!	.INCLUDE	"INC1.H"
!FILE 1 FIRST LINE
FILE:	1 FIRST LINE
!	.INCLUDE	"INC2.H"
!	FILE 2 FIRST LINE
	FILE 2 FIRST LINE
!	FILE 2 LAST LINE
	FILE 2 LAST LINE
!FILE 1 LAST LINE
FILE:	1 LAST LINE
!	THERE
	THERE
!	.END
